### ISSUE TITLE
master branch is broken

### ISSUE THREAD

**Issue by syohex** [konoha-project/libzen#4]
I can't build master branch source code and I got following errors.

```
% make
java version "1.7.0_25"
OpenJDK Runtime Environment (IcedTea 2.3.12) (7u25-2.3.12-4ubuntu3)
OpenJDK 64-Bit Server VM (build 23.7-b01, mixed mode)
Building Java implementation
Buildfile: /home/syohei/Desktop/libzen/build.xml

build:
    [javac] Compiling 25 source files to /home/syohei/Desktop/libzen/build
    [javac] /home/syohei/Desktop/libzen/src/zen/lang/ZenFunc.java:32: error: cannot find symbol
    [javac] public class ZenFunc implements ZenFuncFlag {
    [javac]                                 ^
    [javac]   symbol: class ZenFuncFlag
    [javac] /home/syohei/Desktop/libzen/src/zen/lang/ZenGrammar.java:312: warning: [deprecation] ReportTokenError1(int,String,String) in ZenTokenContext has been deprecated
    [javac]                 TokenContext.ReportTokenError1(ZenLogger.ErrorLevel, "expected \" to close the string literal", LibZen.SubString(SourceText, start, pos));
    [javac]                             ^
    [javac] /home/syohei/Desktop/libzen/src/zen/lang/ZenGrammar.java:317: warning: [deprecation] ReportTokenError1(int,String,String) in ZenTokenContext has been deprecated
    [javac]         TokenContext.ReportTokenError1(ZenLogger.ErrorLevel, "expected \" to close the string literal", LibZen.SubString(SourceText, start, pos));
    [javac]                     ^
    [javac] 1 error
    [javac] 2 warnings

BUILD FAILED
/home/syohei/Desktop/libzen/build.xml:13: Compile failed; see the compiler error output for details.

Total time: 1 second
make: *** ["./libzen.jar"] Error 1
```

**Comment by syohex** [comment]
I think d2835186b50800d612ddb1fc21bb54690598cd37 broke master branch.

**Comment by imasahiro** [comment]
@kkuramitsu  Please append ZenFuncFlag.java

**Comment by syohex** [comment]
This issue was fixed at 9f52742c1e4533e63b89494bd187c412762c6502
